src/DynamoPublish/Models/PublishModel.cs

```diff
@@ -68,18 +135,6 @@ public bool IsLoggedIn
             }
         }
 
-        public HomeWorkspaceModel HomeWorkspace
-        {
-            get;
-            private set;
-        }
-
-        public List<ICustomNodeWorkspaceModel> CustomNodeWorkspaces
-        {
-            get;
-            private set;
-        }
-
```

Comment:
Previously, this state was persisted for testing purposes. Now we use the `WorkspaceDependencies` inner class to hold onto this data and simplify testing. There is no longer a need to create and `InviteModel` to test this behavior.